sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -193,8 +193,8 @@ public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse
             .flux()
             .repeat()
             .takeUntil(pollResponse -> isComplete())
-            .share()
-            .delaySubscription(onActivation);
```

Comment:
What's the difference here?

Reply:
Anu has a good description of the difference in Observation 1
https://github.com/Azure/azure-sdk-for-java/pull/5932#discussion_r336798299